### ISSUE TITLE
The training loss problem for shake

### ISSUE THREAD

**Issue by cloak-s** [lliai/SHAKE#1]
“  elif opt.distill == 'shake':
            loss_kd = criterion_div(pred_feat_s, logit_s.detach()) + criterion_div(logit_s, pred_feat_s.detach())
            loss_kd += criterion_cls(pred_feat_s, target) + F.mse_loss(pred_feat_s, logit_t.detach())” 
Why is there an _**extra learning from proxy to ground-truth_** here, it's different from the way it's implemented in the paper yah.
